app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -94,6 +94,10 @@ def available_resource_pools(opts = {})
       resource_pools({ :cluster_id => opts[:cluster_id] })
     end
 
+    def available_images
+      client.templates
```

Comment:
in my setup, i took
```Completed 200 OK in 34335.1ms (Views: 34322.4ms | ActiveRecord: 6.1ms)```
additionally, turbolinks loads that page twice from some reason, can we change it to ajax instead?